english/vote-the-president/trump/trump.md

```diff
@@ -6,6 +6,8 @@ Are you ready for [WWIII](./WWIII/WWIII.md)?
 
 Know about [Latest hit from Trump Floyd "The Wall"](./trump-floyd-the-wall/thewall.md) !
 
+You must have a tiny.. pair of [hands](https://www.reddit.com/r/gifs/comments/48xv6z/tiny_hands_trump_oc/).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.